### ISSUE TITLE
localizing buttonText for listMonth, listWeek and listYear

### ISSUE THREAD

**Issue by Robrecht** [fullcalendar/fullcalendar#4544]
Hi, currently there is no easy way to localize the buttontext for the list views: listMonth, listWeek and listYear
They are not in the locale files.
My current workaround is to use the localization of my application on the serverside. 
A clean solution would be to add the texts to the locale files.

**Comment by acerix** [comment]
Thanks for the suggestion, even in English the buttons just have "list" as the text. 

https://codepen.io/anon/pen/qvrwRO?editors=0010

You can specify different text using `buttonText` for now, and use something like gettext to localize it.

**Comment by arshaw** [comment]
I agree that this should be easier if one wants to set time-interval text as the text in the list-view buttons. Maybe something like this:

```
locale: 'fr',
buttonText: {
  listWeek: '<week>'
}
```

`<week>` would signify the localized translation for a week. not a great API, i know, i need to think more

**Comment by arshaw** [comment]
this issue goes hand-in-hand with https://github.com/fullcalendar/fullcalendar/issues/4421
